scripts/global.js

```diff
@@ -1824,7 +1797,7 @@ function addProposalToFinalisationCache(cProposal) {
 }
 
 /**
- * Render Governance proposal objects to a given Proposal category
+ * Render Governance proosal objects to a given Proposal category
```

Comment:
Someone stole a **P**

```suggestion
 * Render Governance proposal objects to a given Proposal category
```